src/main/java/net/minestom/server/instance/chunksystem/impl/ChunkAccess.java

```diff
@@ -0,0 +1,14 @@
+package net.minestom.server.instance.chunksystem.impl;
+
+import net.minestom.server.instance.Chunk;
+import org.jetbrains.annotations.ApiStatus;
+
+/**
+ * <b>For internal use only, may be changed at any time. Do not use this.</b>
+ * <p>
+ * An interface for the chunk system to access protected methods of some classes like {@link Chunk}
+ */
+@ApiStatus.Internal
+public interface ChunkAccess {
```

Comment:
Follow up to the above comment - This needs to go. this does nothing but scream terrible api.